package.json

```diff
@@ -98,14 +93,22 @@
   },
   "browserslist": {
     "production": [
-      ">0.2%",
+      ">0.3%",
+      "not IE > 0",
+      "not op_mini all",
       "not dead",
-      "not op_mini all"
+      "last 2 Edge major versions",
```

Comment:
@sambacha Should we really drop de polyfills for almost everything? We will destroy the styling in not updated browsers. How much speed we can get from removing this pollys?

Reply:
Maybe I'm mistaken in how the config works (include / exclude) :)

Reply:
With this config we will have polys for:

>0.3% &&
last 2 major versions (even if they are not in the > 0.3%) &&
Chrome 99 but exclude ios safari 12.5?

If that's the case then I think it's ok and I didn't interpret it ok the first time